src/pages/STG_PanelContacts.page-meta.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexPage xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>37.0</apiVersion>
+    <apiVersion>43.0</apiVersion>
```

Comment:
This appears to be the cause of the styling change. Now just have to figure out the best resolution. Might be to move NPSP Settings to use `<apex:slds/>` instead of the static resource. https://releasenotes.docs.salesforce.com/en-us/winter18/release-notes/rn_vf_other_changes.htm?edition=&impact=

Reply:
@force2b should we keep settings at 37 and tackle that when we tackle the settings slds item? Or do you think it'd be easy to update? We'd also need to fix the dashes...

Reply:
I played around with this yesterday and ended up moving the settings pages over to `<apex:sldsd/>`. I'll create a separate PR for review of that since there were a lot of code changes - styles, images, etc. One thing I did notice is that some of the colors have changed just as part of using Salesforce provided slds. The old version in the static resource was not up to date with the current styling.

Reply:
Awesome, will we want to review and merge the other one first then?

Reply:
Yes. The other PR will be a child of this branch. Hoping to have that to you tomorrow some time. I want to spend some time reviewing the settings pages in detail. Already found some issues with icons.